pkg/base1/cockpit.js

```diff
@@ -3008,8 +3046,11 @@ function full_scope(cockpit, $, po) {
                 }
             }
 
+            for (i = 0; i < fetches.length; i++)
```

Comment:
Could you explain why this change is necessary? Why does calling fetch() interleaved with process() cause problems?